cmd/admin/cert.go

```diff
@@ -234,6 +254,12 @@ func (a *admin) revokeSerials(ctx context.Context, serials []string, reason revo
 		go func() {
 			defer wg.Done()
 			for serial := range work {
+				serial, serialErr := a.checkValidSerial(serial)
```

Comment:
No need to name this error something different, it can just be `err`.

But we should name the result of this function something else, like `cleanedSerial`, so that we're not "shadowing" the loop variable and can still refer to the original in error messages.

```suggestion
				cleanedSerial, err := a.checkValidSerial(serial)
```